DXRMissions/DeusEx/Classes/DXRMissionsM01.uc

```diff
@@ -134,19 +134,16 @@ function MissionTimer()
 
 function AddMissionGoals()
 {
-    local DeusExGoal newGoal;
+    local #var(PlayerPawn) p;
+
     if(dxr.localURL != "01_NYC_UNATCOISLAND") return;
 
     //The MeetPaul conversation would normally give you several goals.
     //Give them manually instead of via that conversation.
-    newGoal=player().AddGoal('DefeatNSFCommandCenter',True);
-    newGoal.SetText("The NSF seem to be directing the attack from somewhere on the island.  Find the commander.");
```

Comment:
you deleted this and replaced it with the goal text from MeetPaul, however that goal text is different

```
The NSF seem to be directing the attack from the Statue.  Get inside and locate the threat.
```

Reply:
Ah. I thought I caught all the goal text that was changed but I missed that one.